docs/src/pages/components/css-baseline/css-baseline.md

```diff
@@ -75,9 +75,11 @@ import darkScrollbar from '@material-ui/core/darkScrollbar';
 const theme = createMuiTheme({
   components: {
     MuiCssBaseline: {
-      styleOverrides: {
-        body: theme.palette.mode === 'dark' ? darkScrollbar() : null,
-      },
+      styleOverrides: `
+        body {
+          ${theme.palette.mode === 'dark' ? darkScrollbar() : ''}
+        }  
+      `,
```

Comment:
Is there something that prevents us to support strings and objects? From what I understand the issue is that 

```jsx
{
 '@font-face': { foo: 'foo' },
 '@font-face': { bar: 'bar }, 
}
```

Doesn't work but with `darkScrollbar`, no issues at all.

Reply:
Ok, one step back. We agree that the issue is that 

```
{ 
  '@font-face': { foo: 'foo' },
  '@font-face': { bar: 'bar },
}
```

won't work simply because Javacript will keep only the last one. If we say that we support only strings for overrides we are solving that issue 👍 

But now that we allow only strings in the overrides, it means that all style function we may invoke in the overrides, must return strings, hence I needed to change the `darkScrollbar()` utility to return string, otherwise this would have been broken.

---

On a different note, we may support as it previously was array for `@font-face` and have a custom `deeepmerge` that would do a concatenation of the arrays instead of replacing them.

Reply:
I can test to see if the alternative method really works and see can decide what would work better. I may be wrong with my assumption here...

Reply:
Sounds about right. I would personally advocate for:

1. Having theme.components.MuiX.stylesOverrides.keu to support strings and objects
2. Update the section about font face to use string as JavaScript is a bottleneck in such a case

Reply:
Agree, let me as a first step update the CssBaseline, and then we can update all style overrides to support strings too in a dedicated PR.